llvm/tools/file-table-tform/file-table-tform.cpp

```diff
@@ -78,13 +78,19 @@ static cl::opt<std::string> Output("o", cl::Required,
                                    cl::cat(FileTableTformCat));
 
 static constexpr char OPT_REPLACE[] = "replace";
+static constexpr char OPT_REPLACE_CELL[] = "replace_cell";
 static constexpr char OPT_RENAME[] = "rename";
 static constexpr char OPT_EXTRACT[] = "extract";
 
 static cl::list<std::string> TformReplace{
     OPT_REPLACE, cl::ZeroOrMore, cl::desc("replace a column"),
     cl::value_desc("<column name or ordinal>"), cl::cat(FileTableTformCat)};
 
+static cl::list<std::string> TformReplaceCell{
```

Comment:
We should probably add tests for those two new actions performed by the tool

Reply:
I have added llvm/test/tools/file-table-tform/file-table-tform-single.test and added a test case in llvm/unittests/Support/SimpleTableTest.cpp.